test/performance/odp_ipsec.c

```diff
@@ -971,8 +1031,18 @@ static void parse_args(int argc, char *argv[], ipsec_args_t *cargs)
 		case 'd':
 			cargs->debug_packets = 1;
 			break;
-		case 'i':
-			cargs->iteration_count = atoi(optarg);
+		case 'c':
```

Comment:
Now that the argument letter changes, odp_ipsec_run.sh needs an update too.

Reply:
Updated in v3.